docsearch/algolia_configs/latest_commit.json

```diff
@@ -26,6 +27,7 @@
     "en-tidb-v3.0": "ff3bdb392d243f7a48051e037fb750579e7a6ca5",
     "en-tidb-v2.1": "2755565832044b9a1fdc4be01057dc398098e6f3",
     "zh-tidb-dev": "f27485b695f5955aac3922b4b5e908d0950ca854",
+    "zh-tidb-v8.2": "to-be-updated",
```

Comment:
to-be-updated according to the latest commit in https://github.com/pingcap/docs/commits/release-8.2/